faraday-my_adapter.gemspec

```diff
@@ -1,27 +1,29 @@
 # frozen_string_literal: true
 
-require_relative 'lib/faraday/my_adapter/version'
+require_relative 'lib/faraday/em_http/version'
 
 Gem::Specification.new do |spec|
-  spec.name = 'faraday-my_adapter'
-  spec.version = Faraday::MyAdapter::VERSION
-  spec.authors = ['Your Name']
-  spec.email = ['your_name@gmail.com']
-
-  spec.summary = 'Faraday adapter for MyAdapter'
-  spec.description = 'Faraday adapter for MyAdapter'
-  spec.homepage = 'https://github.com/lostisland/faraday-my_adapter'
+  spec.name = 'faraday-em_http'
+  spec.version = Faraday::EmHttp::VERSION
+  spec.authors = ['@iMacTia']
+  spec.email = ['giuffrida.mattia@gmail.com']
+
+  spec.summary = 'Faraday adapter for Em::Http'
+  spec.description = 'Faraday adapter for Em::Http'
+  spec.homepage = 'https://github.com/lostisland/faraday-em_http'
   spec.license = 'MIT'
 
   spec.required_ruby_version = Gem::Requirement.new('>= 2.4.0')
 
   spec.metadata['homepage_uri'] = spec.homepage
-  spec.metadata['source_code_uri'] = 'https://github.com/lostisland/faraday-my_adapter'
-  spec.metadata['changelog_uri'] = 'https://github.com/lostisland/faraday-my_adapter'
+  spec.metadata['source_code_uri'] = 'https://github.com/lostisland/faraday-em_http'
+  spec.metadata['changelog_uri'] = 'https://github.com/lostisland/faraday-em_http'
 
   spec.files = Dir.glob('lib/**/*') + %w[README.md LICENSE.md]
   spec.require_paths = ['lib']
 
+  # TODO: make this a normal dependency when releasing v2.0
```

Comment:
When releasing Faraday 2.0

That's perhaps a note more people with less context can get.

Reply:
I actually mean v2.0 of this gem, but that will happen together with Faraday 2.0 as well.
I agree it makes sense to point that out 👍